tools/build_env/build_env.sh

```diff
@@ -40,18 +43,18 @@ if ! which docker > /dev/null; then
 fi
 
 if [[ $BUILD = true ]]; then
-  readonly DOCKER_ROOT="$(dirname $0)"
-  docker build --rm -t $IMAGE_NAME $DOCKER_ROOT
+  readonly DOCKER_ROOT="$(dirname $0)/$TAG"
```

Comment:
Sorry, my latest change does just that: `uproxy/build-default` and `uproxy/build-android`. No tags.

Specifically, I think `uproxy/build-default` is a little inelegant...WDYT about making the default image be `uproxy/build` and variants take the form `uproxy/build-android`?